ambry-api/src/test/java/com.github.ambry/rest/RestUtilsTest.java

```diff
@@ -666,4 +669,24 @@ public static void setUserMetadataHeaders(JSONObject headers, Map<String, String
       headers.put(key, userMetadata.get(key));
     }
   }
+
+  @Test
+  public void toSecondsPrecisionInMsTest() {
+    assertEquals(0, RestUtils.toSecondsPrecisionInMs(999));
+    assertEquals(1000, RestUtils.toSecondsPrecisionInMs(1000));
+    assertEquals(1000, RestUtils.toSecondsPrecisionInMs(1001));
+  }
+
+  @Test
+  public void getTimeFromDateStringTest() {
+    SimpleDateFormat dateFormatter = new SimpleDateFormat(RestUtils.HTTP_DATE_FORMAT, Locale.US);
+    long curTime = System.currentTimeMillis();
+    Date curDate = new Date(curTime);
+    String dateStr = dateFormatter.format(curDate);
+    long epochTime = RestUtils.getTimeFromDateString(dateStr);
+    long actualExpectedTime = (curTime / 1000L) * 1000;
+    // Note http time is kept in Seconds so last three digits will be 000
+    assertEquals("Time mismatch ", actualExpectedTime, epochTime);
+    assertEquals("Should have returned null", null, RestUtils.getTimeFromDateString("abc"));
+  }
```

Comment:
add another test here where you create a time that is not `Locale.US` and check the output.